patches/tags_pointer_type_fix.patch

```diff
@@ -0,0 +1,11 @@
+--- agg2/font_freetype/agg_font_freetype.cpp.orig	2024-03-06 10:16:58
```

Comment:
I just want to make sure the inclusion of this file is intentional

Reply:
Yes. I 'd like to start documenting patches made to the vendored agg code so that we can potentially patch them in the future. I think in another PR or another branch or maybe just an email thread with an outside contributor I have documented what changes have been done so far. But from now on I'd really like to document them better. I've updated the README to talk about these patches.